src/lib/docs.ts

```diff
@@ -22,15 +27,12 @@ import {
   isFunctionDeclaration,
   isMethodDeclaration,
   isModuleDeclaration,
-  isVariableStatement,
-  ModifierFlags,
-  ModuleKind,
-  NodeFlags,
-  ScriptTarget,
-  SyntaxKind
+  isVariableStatement
 } from 'typescript';
 import type {BuildOptions, DocEntry, DocEntryConstructor, DocEntryType} from './types';
 
+const {relative, resolve} = posix;
```

Comment:
Nitpick: If only used in a single function, I would not deconstruct those at the module root but, within the function where those are used.

Reply:
gotcha, will update